src/main.rs

```diff
@@ -1,14 +0,0 @@
-#![cfg_attr(feature = "strict", deny(warnings))]
```

Comment:
could we keep this as an [example](http://doc.crates.io/guide.html#project-layout)? it's pretty convenient for testing